src/Services/Generators/Laravolt.php

```diff
@@ -5,22 +5,45 @@
 use Illuminate\Http\File;
 use Illuminate\Support\Collection;
 use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Str;
+use LaravelEnso\Avatars\Enums\Types;
 use LaravelEnso\Avatars\Models\Avatar;
 use Laravolt\Avatar\Facade as Service;
 
 class Laravolt extends Generator
 {
     private const FontSize = 128;
+    private const Filename = 'avatar';
+    private const Extension = 'jpg';
+    private string $path;
 
-    public function generate(): File
+    public function handle(): ?Avatar
+    {
+        $this->generate()->attach();
```

Comment:
let's do generate->persist->attach

right now attach also persists the avatar model, extract it